keyboards/pikatea/pikatea_macropad_gb4/readme.md

```diff
@@ -0,0 +1,19 @@
+# pikatea/pikatea_macropad_gb4
+
+![pikatea](https://i.imgur.com/UpcSHLi.png)
+
+*3, 5 or 7 keys and a knob mounted to an underside surface*
+
+* Keyboard Maintainer: [Jack Kester](https://github.com/JackPikatea)
+* Hardware Supported: *M32u4*
+* Hardware Availability: *http://www.pikatea.com*
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```